TransformationSystem/Client/test/Test_Client_TransformationSystem.py

```diff
@@ -12,6 +12,21 @@
 
 #############################################################################
 
+
+class reqValFake_C(object):
```

Comment:
Sincerely, what you ask is a more complex way of what I've done and 
would lead to the same result. This is not the test of the 
RequestValidator, that should have its own test.